src/AutoRest.CSharp/Common/Output/Models/Types/SystemObjectType.cs

```diff
@@ -215,7 +216,8 @@ private string GetPropertySummary(MethodInfo? setter)
             return setter != null ? " or sets" : string.Empty;
         }
 
-        protected override ObjectTypeConstructor BuildSerializationConstructor() => BuildConstructor(GetCtor(_type, ReferenceClassFinder.SerializationCtorAttributeName));
+        protected override ObjectTypeConstructor BuildSerializationConstructor()
```

Comment:
There was a bug when `SystemObjectType` has been using initialization constructor as a base constructor instead of serialization one.